core/src/integration-test/java/com/scalar/db/api/DistributedStorageIntegrationTestBase.java

```diff
@@ -95,6 +95,10 @@ private void createTable() throws ExecutionException {
         options);
   }
 
+  protected int getLargeDataSizeInBytes() {
+    return 5000;
+  }
```

Comment:
Since the maximum blob size is limited to 2000 bytes for Oracle, I had to create an exception here.